### PR TITLE
fix hardcoded S3 Bucket to backup ETCD to

### PR DESCRIPTION
**Pull request by szuecs**
Fix for #89 
Default is set in cluster.py and there is a new parameter for senza.

### CODE REVIEW

File: cluster/cluster.py [hunk 1]
```diff
@@ -231,6 +232,7 @@ def cli():
 @click.option('--instance-type', type=str, default='t2.micro', help='Type of instance')
 @click.option('--master-nodes', default=1, type=int, help='Number of master nodes')
 @click.option('--worker-nodes', default=1, type=int, help='Number of worker nodes')
+@click.option('--etcd_s3_bucket', default='teapot-etcd-backup', help='S3 Bucket to backup ETCD to')
```
Comment (mikkeloscar) [inline cluster/cluster.py]: Is `teapot-etcd-backup` the best default name? Maybe something like `k8s-etcd-backup` or `cluster-etcd-backup` is better?

Comment (hjacobs) [inline cluster/cluster.py]: please use `--etcd-s3-bucket` as option name to be consistent

Comment (szuecs) [inline cluster/cluster.py]: The problem is that a bucket has a global name, so a custom prefix makes sense to have.

Comment (hjacobs) [inline cluster/cluster.py]: S3 bucket names are globally unique, so a default value actually makes no sense

Comment (szuecs) [inline cluster/cluster.py]: done

**Comment by hjacobs** [comment]
Let's make the backup work first, I would afterwards change the script to create a bucket for the account if needed so we need no option.

**Comment by hjacobs** [comment]
I'll quickly do the change...

**Comment by hjacobs** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**